app/lib/dqt/params/trn_request.rb

```diff
@@ -112,6 +112,7 @@ def qualification_params
           "subject" => subject_code,
           "class" => DEGREE_CLASSES[degree.grade],
           "date" => Date.parse("01-01-#{degree.graduation_year}").iso8601,
+          "heQualificationType" => CodeSets::DegreeTypes::MAPPING[degree.uk_degree_uuid],
```

Comment:
Does this need a `.to_s` in case a nil is passed? Maybe we could add a test case for it.

Reply:
Updated and test added!